Cargo.toml

```diff
@@ -55,7 +55,7 @@ wayland = ["bevy_winit/wayland"]
 x11 = ["bevy_winit/x11"]
 
 [workspace]
-members = ["crates/*", "crates/bevy_ecs/hecs"]
+members = ["crates/*", "crates/bevy_ecs/hecs", "examples/ios"]
```

Comment:
Because this example is a full cargo project it's gotta be added to the workspace. Cargo gets angry if it's in a sub directory of a workspace without being added too it. I find it a bit annoying but think there's probably a good reason why the cargo folks did that.